src/python/WMCore/Storage/Execute.py

```diff
@@ -21,7 +22,10 @@ def runCommand(command):
 
     """
     # capture stdout and stderr from command
-    child = Popen(command, shell=True, bufsize=1, stdin=PIPE, close_fds=True)
+    if PY3:
+        child = Popen(command, shell=True, bufsize=1, stdin=PIPE, close_fds=True, encoding='utf8')
```

Comment:
Just a note: pylint2 complains with:
```
E1123: Unexpected keyword argument 'encoding' in constructor call
```

To me, it's fine like this. But if you want to add a comment to at least one of these commands, that would be fine either.

Reply:
yeah, the python2 pylint does not skip the code under the `if PY3`. I will add a comment in the code, thanks for the suggestion!